.circleci/config.yml

```diff
@@ -84,13 +98,25 @@ commands:
       - scala-build
 
 jobs:
-  run-tests:
+  run-test-groupA:
     executor: main-env
     steps:
       - repo-setup
-      - firesimtest-manual-midasexamples
+      - run-scala-test:
+          test-name: "CIGroupA"
       - build-scala-doc
 
+  run-test-groupB:
+    executor: main-env
+    steps:
+      - repo-setup
+      - run-scala-test:
+          test-name: "CIGroupB"
+      - run-scala-test:
```

Comment:
So if I understand this correctly, this will run all tests under midas?

Reply:
Correct.